relay-general/src/pii/legacy.rs

```diff
@@ -76,6 +76,9 @@ impl DataScrubbingConfig {
     /// Like [`pii_config`](Self::pii_config) but without internal caching.
     #[inline]
     pub fn pii_config_uncached(&self) -> Result<Option<PiiConfig>, PiiConfigError> {
-        convert::to_pii_config(self)
+        convert::to_pii_config(self).map_err(|e| {
+            relay_log::error!("Failed to convert datascrubbing config");
```

Comment:
We add a project tag to the Sentry scope in `EnvelopeProcessor::process()`, so this error should be tagged by it as well.

https://github.com/getsentry/relay/blob/712861d26abe041be4821285b67d34e5e1fa2c08/relay-server/src/actors/processor.rs#L2023-L2031